apps/utils/tests/conftest.py

```diff
@@ -12,3 +12,40 @@
 def _setup_test_app_fixture():
     yield
     tear_down_test_app(app_label)
+
+
+@pytest.fixture()
+def _model_setup():
+    """
+    Setup models for testing deletion utilities.
+
+    c1
+        -> b1
+            -> t1 -> p1, p2
+            -> t2
+        -> b2 -> t1 -> p1, p2
+        -> t1 -> p1, p2
+        -> t2
+    c2
+        -> t2
+        -> b3
+    t3
```

Comment:
`t3` should be `-> t3`?

Reply:
no because t3 isn't linked to any collection or bot

Reply:
My bad, cool cool